### DISCUSSION TITLE
Idee inerenti alla funzione d'inserimento assenze multiple.

### DISCUSSION THREAD

**Discussion by gianlucaaiello56** [iisgiua/giuaschool#506] [Idee e proposte]
Salve @trinko ,
volevo proporre l'inserimento della funzione "inserisci evento multiplo" ad esempio inserire la nuova modalità FUORICLASSE  a tutti gli studenti.
Cordialmente,
Aiello Gianluca 
(v.d.s. liceo "Re Vittorio Emanuele" istituto il quale utilizza giua@school)

**Comment by trinko** [comment]
A mio avviso il FUORI CLASSE ha un significato diverso. Si usa quando qualche alunno della classe è presente ma svolge un'attività al di fuori della classe e quindi non è sotto la responsabilità del docente che firma le lezioni.
Il caso dell'intera classe che svolge un'attività al di fuori della scuola (gita, uscita didattica, ecc.) è sotto la responsabilità del docente che firma la lezione, anche se questa attività è al di fuori della scuola.

**Comment by trinko** [comment]
E' anche vero che ho visto anch'io su altri registri ad utilizzare il FUORI CLASSE per le gite scolastiche e le uscite didattiche. 
Per cui, in sostanza, penso che per l'anno prossimo farò una revisione della gestione del fuori classe.
